Components/Navigation/navigation.html

```diff
@@ -0,0 +1,97 @@
+        <section class="navigation-section">
+            <div class="section-title">
+                Navigation
+            </div>
+            <div class="section-subtitle">
+              
+            </div>
+           
+            <article>
+               
+                <div class=" nav-example-div">
+                    <div class="navigation-container">
+                        <div class="nav-header">
+                            <h1>Handy UI</h1>
+                        </div>
+                        <nav class="main-links">
+                            <a href="https://github.com/kotesh-arya"><img class="header-icon"
+                                    src="https://img.icons8.com/windows/64/ffffff/github.png" /></a>
+                            <a href="https://twitter.com/Kotesh_Mudila"><img class="header-icon"
+                                    src="https://img.icons8.com/ios-glyphs/60/ffffff/twitter--v1.png" /></a>
+                            <a href="https://www.linkedin.com/in/koteswara-rao-mudila-7aa740159/"><img class="header-icon"
+                                    src="https://img.icons8.com/ios-glyphs/60/ffffff/linkedin.png" /></a>
+                            </nav>
+                    </div>
+                </div>
+                <div class="code-frame">
+                    <iframe class="frame-div"
+                    src="https://carbon.now.sh/embed?bg=rgba%28184%2C176%2C141%2C0%29&t=3024-night&wt=none&l=auto&width=680&ds=true&dsyoff=0px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=4x&wm=false&code=%253Cdiv%2520class%253D%2522navigation%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522nav-header%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch1%253EHandy%2520UI%253C%252Fh1%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cnav%2520class%253D%2522main-links%2522%253E%250A%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2522%253E%253Cimg%2520class%253D%2522header-icon%2522src%253D%2522https%253A%252F%252Fimg.icons8.com%252F%252Fgithub.png%2522%2520%252F%253E%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2522%253E%253Cimg%2520class%253D%2522header-icon%2522src%253D%2522https%253A%252F%252Fimg.icons8.com%252F%252Ftwitter--v1.png%2522%252F%253E%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2522%253E%253Cimg%2520class%253D%2522header-icon%2522src%253D%2522https%253A%252F%252Fimg.icons8.com%252Flinkedin.png%2522%2520%252F%253E%253C%252Fa%253E%250A%2520%2520%2520%2520%253C%252Fnav%253E%250A%253C%252Fdiv%253E"
+                    sandbox="allow-scripts allow-same-origin">
+                  </iframe>
+                </div>
+            </article>
+           
+            <div class="section-footer">
+                <a href="/Components/Grid/grid.html">
+                    < Grid</a>
```

Comment:
check here.

Reply:
That's actually left arrow option for the previous page